### ISSUE TITLE
Telegram Connector: Fix update interval code

### ISSUE THREAD

**Issue by FabioRosado** [opsdroid/opsdroid#996]
**This issue is aimed at folks that wish to get started contributing to open source**

Today I was setting up opsdroid on a raspberry pi which will be running continuously so I had to change the update interval on the configuration to a bit longer than 1 second.

I have noticed that there is an issue with the update interval code. 

In the documentation it says that we need to set update interval like this: `update-interval: 1` but on the Telegram connector code we are trying to get `update_interval` from the config. This should be changed to use the one in the documentation.

Basically, we need to change [this line](https://github.com/opsdroid/opsdroid/blob/9cd7b3669e5328e28cc6555fde87b4c8c6091903/opsdroid/connector/telegram/__init__.py#L33)

**Comment by athanikos** [comment]
Hello,
I just started working  on opsdroid. Could I work on this issue?
Thanks 
Nikos

**Comment by FabioRosado** [comment]
Heya yeah that will be great! Let me know if you need any help with anything 😄👍
